### PR TITLE
UIPFPAT 51: add axe accessibility tests

### PR DESCRIPTION
**Pull request by daniil-patckin**
Add axe accessibility tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "regenerator-runtime": "^0.13.3"
   },
   "dependencies": {
+    "@folio/stripes-testing": "^4.5.0",
```
Comment (BogdanDenis) [inline package.json:64]: This should go into devDeps since it's not required for actual module functionality

**Review by BogdanDenis — CHANGES_REQUESTED** [review]
Please add a CHANGELOG entry describing the change
Also, branches should only be named with ticket numbers: `UIPFPAT-51`

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 ## [5.0.0] (IN PROGRESS)
 
 * bump stripes to 8.0.0 for Orchid/2023-R1. Refs UIPFPAT-52.
+* Add a11y tests
```
Comment (Dmytro-Melnyshyn) [inline CHANGELOG.md:5]: Please add `Refs UIPFPAT-51` to the end.
`Add a11y tests. Refs UIPFPAT-51.`

Comment (daniil-patckin) [reply CHANGELOG.md]: Done

**Review by BogdanDenis — APPROVED** [review]

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Pull request merged**